llama_stack/cli/stack/build.py

```diff
@@ -74,6 +74,13 @@ def _add_arguments(self):
             default=False,
             help="Run the stack after building using the same image type, name, and other applicable arguments",
         )
+        self.parser.add_argument(
+            "--providers",
+            type=str,
+            default=None,
+            nargs="+",
```

Comment:
why `+`? if we allow for the format I suggested, could this be just a single argument?

Reply:
sure, I was thinking it would be:

`--providers api1=provider1 api1=provider2 api2=provider3` rather than `--providers api1=provider1,api1=provider2,api2=provider3`

But can switch it to the `,` version instead